wodles/aws/aws_s3.py

```diff
@@ -183,6 +183,20 @@ def main(argv):
                     discard_regex=options.discard_regex,
                     bucket_handler=bucket_handler,
                     message_processor=subscribers.sqs_message_processor.AWSS3MessageProcessor)
+            elif options.subscriber.lower() == "security_hub":
+                bucket_handler = subscribers.s3_log_handler.AWSSecurityHubSubscriberBucket
+                asl_queue = subscribers.sqs_queue.AWSSQSQueue(
```

Comment:
This variable's name is a bit confusing, I'm interpreting it as `Amazon Security Lake Queue` but the queue is used by other services as well, not only security lake. I would change this and the other declarations to `sqs_queue` or simply `queue`.

```suggestion
                sqs_queue = subscribers.sqs_queue.AWSSQSQueue(
```

Reply:
I agree. Renamed to `subscriber_queue`